java/code/src/com/redhat/rhn/taskomatic/TaskoXmlRpcHandler.java

```diff
@@ -644,7 +644,6 @@ private void isAlreadyScheduled(Integer orgId, String jobLabel)
      * @return true if JMX system props props
      */
     public boolean isJmxEnabled() {
-        return StringUtils.isNotEmpty(System.getProperty("com.sun.management.jmxremote.port")) &&
-                StringUtils.isNotEmpty(System.getProperty("java.rmi.server.hostname"));
+        return StringUtils.contains(System.getenv("JAVA_OPTS"), "jmx_prometheus_javaagent.jar");
```

Comment:
I do not really like the fact that `/etc/rhn/taskomatic.conf` is actually interpreted as a shell script.

I understand that is none of your fault, but I would still ask for a way not to build on top of that.

Can we check for the presence of the necessary classes via the class loader (eg. `Class.forName`) instead of using this environment variable?